config/solr.yml

```diff
@@ -1,2 +1,6 @@
- production:
+development:
+  url: http://127.0.0.1:<%= ENV['SOLR_TEST_PORT'] || 8983 %>/solr/aster-development
+test:
+  url: http://127.0.0.1:<%= ENV['SOLR_TEST_PORT'] || 8985 %>/solr/aster-test
+production:
```

Comment:
Instead of hard-coding pieces of the URL, could we allow the entire URL to be configurable?  Something like this:

```
development:
  adapter: solr
  url: <%= ENV['DEV_SOLR_URL'] || "http://127.0.0.1:8983/solr/aster-development" %>
test: &test
  adapter: solr
  url: <%= ENV['TEST_SOLR_URL'] || 'http://127.0.0.1:8985/solr/aster-test' %>
```